src/uci.cpp

```diff
@@ -311,8 +311,14 @@ string UCI::value(Value v) {
 
   stringstream ss;
 
+  // Formerly PawnValueEg. This value currently results in 100cp having a 50% win rate.
+  // This is not too different from the historical value (113cp was 50% win rate in 2020)
+  // It can be obtained (and should be synced) with the win_rate_model, it is the value
+  // of the 'a' parameter at move 32 (which is the sum of the parameter array for a).
```

Comment:
I like this PR very much. However, these comments should be made explicit and enforced at compile time.

Here is a suggestion how to do this: https://github.com/ddobbelaere/Stockfish/commit/49de6b694126a46eb191702b9f1bd7638322ea83
It boils down to adding `static_assert(win_rate_model(Value{Internal2Pawn}, 64) == 500);` and making `int win_rate_model` `constexpr`. The comments can be adapted, as in this place, we should not really care how it is calculated (the 'a' parameter is an internal detail of the WDL model).

Reply:
this looks good to me, let me include that code.

Reply:
unfortunately, std::exp is not constexpr for clang... however, can you come up with a patch that asserts this value is equal to the sum of the as ?